src/Mjolnir.ts

```diff
@@ -1023,9 +1023,9 @@ export class Mjolnir {
         return await this.eventRedactionQueue.process(this, roomId);
     }
 
-    private async handleReport(roomId: string, reporterId: string, event: any, reason?: string) {
-        for (const protection of this.enabledProtections) {
-            await protection.handleReport(this, roomId, reporterId, event, reason);
+    private async handleReport(e: any) {
```

Comment:
Please avoid replacing something that is (somewhat) reasonably typed by a `any`, this makes code harder to maintain.

Reply:
Hmm okay, but I'm not sure if the event object has an explicit fixed type cast there for the dictionary.